pkg/operators/machine-api-operator.go

```diff
@@ -164,3 +167,71 @@ var _ = Describe("[Feature:Operators] Machine API cluster operator status should
 		Expect(framework.IsStatusAvailable(client, "machine-api")).To(BeTrue())
 	})
 })
+
+var _ = Describe("[Feature:Operators] When cluster-wide proxy is configured, Machine API cluster operator should ", func() {
```

Comment:
I don't think this can run in parallel with other tests, I think we need to mark as serial as suggested below, but I think @elmiko can confirm this
```suggestion
var _ = Describe("[Serial][Feature:Operators] When cluster-wide proxy is configured, Machine API cluster operator should ", func() {
```